main/zoe/api/zoe-contract-facet.md

```diff
@@ -2,25 +2,23 @@
 
 <Zoe-Version/>
 
-A Zoe Contract Facet is an API object for a running contract instance to access the Zoe state for that instance. A Zoe Contract Facet is accessed synchronously from within the contract, and usually is referred to in code as `zcf`. The contract instance is launched by `E(zoe).makeInstance`, and is given access to the `zcf` object during that launch. In the operation below, the `instanceHandle` is the handle for the running contract instance.
+A Zoe Contract Facet is an API object for a running contract instance. It accesses the instance's Zoe state. From within the contract, you synchronously access a Zoe Contract Facet (usually is referred to in code as `zcf`). The contract instance is launched and given access to the zcf object by `E(zoe).makeInstance`. In the operation below, the `instanceHandle` is the handle for the running contract instance.
+**tyg todo: There is no other instance of `instanceHandle` in this doc. Delete or rework?**
+
+## zcf.reallocate(offerHandles, newAmountKeywordRecords)
 
-## zcf.reallocate(offerHandles, newAmountKeywordRecords, sparseKeywords)
 - `offerHandles` <router-link to="/glossary/#handle">`{Array <Handle>}`</router-link>
 - `newAmountKeywordRecords` <router-link to="/zoe/api/records.html#amountkeywordrecord">`{Array <AmountKeywordRecord>}`</router-link>
-- `sparseKeywords` `{Array <String>}` sparseKeywords is an optional array of string keywords, which may be a subset of allKeywords.
 
-Instruct Zoe to try to reallocate payouts for the given `offerHandles`.  This will only succeed if the reallocation 1) conserves rights, and 2) is 'offer-safe' for all parties involved. This reallocation is partial, meaning that it applies only to
-the amount associated with the offerHandles that are passed in.  We are able to ensure that with each reallocation,
-rights are conserved and offer safety is enforced for all offers, even though the reallocation is partial, because once
-these invariants are true, they will remain true until changes are made.
+Instructs Zoe to try to reallocate payouts for the given `offerHandles`. This only succeeds if the reallocation:
+- Conserves rights.
+- Is 'offer-safe' for all involved parties.
+This reallocation is partial, so it only applies to the amounts associated with passed in `offerHandles`. We ensure that with each reallocation, rights are conserved and offer safety enforced for all offers. This is true even for partial reallocations. Once these invariants are true, they stay true until changes happen.
```

Comment:
Rather than saying that the reallocation is partial, I think the wording that we are moving towards is that the reallocation provided will be used to update the current allocation. The exact mechanism is a merge where the new information overwrites the old if there any overlap. That wording probably isn't the best, so feel free to change it, but I hope that gives an idea of where we want to head.